spec/mxmnd_spec.rb

```diff
@@ -1,7 +1,64 @@
-require "spec_helper"
+require 'spec_helper'
 
 describe Mxmnd do
   it 'raises BadRequest on missing IP address' do
-    expect { Mxmnd::city(nil) }.to raise_error Mxmnd::BadRequest
+    expect { Mxmnd.city(nil) }.to raise_error Mxmnd::BadRequest
+  end
+end
+
+describe 'Mxmnd.city response' do
+  before {
+  	url = 'https://geoip.maxmind.com/geoip/v2.1/city/45.17.22.11'
+    path = './spec/features/mxmnd_res.txt'
+    ruby_headers = {
+      # got from rails hint
+      'Accept' => '*/*',
+      'Accept-Encoding' => 'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
+       'User-Agent' => 'Ruby'
+    }
+    # i've changed "Content-Length" property of recorded response from 1476 to 2515
+    # because of by unknown reason stub didn't return the whole response
+    response_file = File.new(path)
+    stub_request(:get, 'http://https//geoip.maxmind.com/geoip/v2.1/city/45.17.22.11:80/').
+      with(headers: ruby_headers).
+      to_return(response_file)
+    @response = JSON.parse(Net::HTTP.get(url, '/'))
```

Comment:
the use of instance variables in specs is discouraged, Please use `let` for this.
e.g.
```
let(:response) { JSON.parse... }
```

Then you can just use `response` variable inside your spec examples.